gm4_liquid_minecarts/data/liquid_minecarts/functions/init_tank.mcfunction

```diff
@@ -2,7 +2,7 @@
 #run from try_to_unload
 
 data modify block ~ ~ ~ CustomName set from entity @s CustomName
-summon armor_stand ~ ~-.45 ~ {CustomName:"\"gm4_liquid_tank_display\"",Tags:["gm4_no_edit","gm4_liquid_tank_display","gm4_lm_needs_texture"],NoGravity:1,Marker:1,Invisible:1,Invulnerable:1,Small:1,DisabledSlots:2039583,Fire:20000}
+summon armor_stand ~ ~-.45 ~ {CustomName:"\"gm4_liquid_tank_display\"",Tags:["gm4_no_edit","gm4_liquid_tank_display","gm4_lm_needs_texture"],NoGravity:1,Marker:1,Invisible:1,Invulnerable:1,Small:1,DisabledSlots:4144959,Fire:20000}

```

Comment:
Why doesn't the value `2039583` work here? I was pretty sure that `2039583` locks all slot interactions.

Reply:
I just used that value as that is the one i normal used, after testing with the other one that looks to work too, i can revert it to that one if you want?

Reply:
It would be more consistent to revert, sry :/

Reply:
The wiki doesn't say, but the value @kruthers uses it might lock the offhand.